lib/srv/alpnproxy/proxy.go

```diff
@@ -58,6 +60,12 @@ type ProxyConfig struct {
 	// of the connection). It is set to defaults.HandshakeReadDeadline if
 	// unspecified.
 	ReadDeadline time.Duration
+	// IdentityTLSConfig is TLS cluster configuration.
+	IdentityTLSConfig *tls.Config
```

Comment:
It's not super clear what "identity TLS config" is, could you rename the field and/or expand the comment?